__std__/nix/plutus/library/default.nix

```diff
@@ -0,0 +1,8 @@
+{ inputs, cell }@block:
+{
+  plutus-project = import ./plutus-project.nix block;
+
+  agda-project = import ./agda-project.nix block;
+
+  agda-packages = import ./agda-packages.nix block;
```

Comment:
See, in many ways it seems like this should be in `toolchain`, but then it can't be because it depends on `plutus`. Not sure what to say about that, other than it feels like the `toolchain`/`plutus` distinction is hard to keep clear.